payment-menu/capture.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Swedbank Pay Payment Menu – Capture
+sidebar:
+  navigation:
+  - title: Payment Menu
+    items:
+    - url: /payment-menu/
+      title: Introduction
+    - url: /payment-menu/payment-order
+      title: Payment Order
+    - url: /payment-menu/capture
+      title: Capture 
+    - url: /payment-menu/after-payment
+      title: After Payment
+    - url: /payment-menu/other-features
+      title: Other Features
+---
+
+{% include payment-order-capture.md documentation_section="payment-menu"%}
+
+**Et voilà!** The payment should now be complete and
+everyone should be happy. But, sometimes you also need to implement the
+cancellation and reversal operations described below.
```

Comment:
The cancellation and reversal operations aren't described below, but on another page?